internal/command/container.go

```diff
@@ -285,6 +299,28 @@ func fynePackage(ctx Context, image containerImage) error {
 	return nil
 }
 
+func calculateExeName(sourceDir, os string) string {
```

Comment:
What about adding a doc comment explaining that it is ported from the fyne base code and why?
It'd useful for reference keeping track of the reason we have some logic regarding windows even if has impact only for darwin.

Reply:
Great idea